src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -81,7 +87,7 @@ public TurretSubsystem(RobotStateManager robotStateManager) {
       turretSim =
           new SingleJointedArmSim(
               DCMotor.getNEO(1),
-              4,
+              Constants.TurretConstants.TURRET_MOTOR_TURRET_RATIO,
```

Comment:
```suggestion
              1/Constants.TurretConstants.TURRET_MOTOR_TURRET_RATIO,
```
Assuming we want a reduction here, right?